### PR TITLE
Add subscribe argument

### PR DESCRIPTION
**Pull request by yuzhakovvv**
**Problem**
Sometimes feature is too complicated to show or hide it "on the fly". It may require an app reload. For this case we don't want to lister for the `change` event

**Solution**
Add `subscribe` argument to decide whether subscribe to feature flag changes or not. Default value is `true`

### CODE REVIEW

**Comment by yuzhakovvv** [comment]
@yusinto could you check this one?

**Comment by yusinto** [comment]
@yuzhakovvv the way you respond to flag changes is completely up to you. You can detect the flag change via props and then force a reload. ld-redux's job is just to ensure that your app has the correct set of flags. Does this make sense?

**Comment by yuzhakovvv** [comment]
@yusinto the case is not to force a reload on change or control the change somehow. It might affect UX for large apps when user in the middle of something and app just reloads. Instead we want to not subscribe at all.

LaunchDarkly provides subscription as an additional feature, which shows that there are cases when devs may not want to subscribe. Subscription requires much more control of the app's logic to make sure that feature may be turned on/off "on the fly". Do you see what I mean?

**Comment by yuzhakovvv** [comment]
@yusinto any thoughts here? :)

**Comment by mlund01** [comment]
@yuzhakovvv I think what @yusinto is saying is that you can deal with the decision to update your app based on flag changes in something like a `componentShouldUpdate` handler instead of globally unsubscribing from changes. This will prove more flexible and still give you the functionality that you need for flags that shouldn't change on the fly.

Here is an example of how we did it in `ld-redux-components` (a component helper library based on `ld-redux`), https://github.com/wheniwork/ld-redux-components/blob/master/src/components/Feature.js#L24 

Basically, what we did was check if the existing `isLDReady` prop value is set to true or false. If it is false, it means that it is on the initial load or first load from ld servers, so we update. If it is true, it means that it's an "on the fly" flag change, and we skip updating the component if so (only if our `liveUpdate` prop is set to `false`)

**Comment by yuzhakovvv** [comment]
@mlund01 this is a nice workaround, so thanks for the example. But app still receives updates through network and in your case they are useless.

Launch Darkly has feature to not subscribe to changes, so why can't we add it to this awesome package? 😃

**Comment by yusinto** [comment]
@yuzhakovvv ok after thinking about it I think this change makes sense for complex UI which does not want to subscribe to changes. Are you able to write unit tests for this change though please?

**Comment by yuzhakovvv** [comment]
@yusinto actually I tried from the begging but unfortunately didn't figure out the syntax. If it easy to you, could you cover this?

**Comment by yuzhakovvv** [comment]
@yusinto so could you cover this?

**Comment by yusinto** [comment]
Apologies for the delay. I have added the subscribe option [here](https://github.com/yusinto/ld-redux/commit/5e661c1b2f3bd4f5488d4fb62f487a5f5e38cdc0) with tests. It's published, the version you want should be v3.2.4.
